compiler/ballerina-lang/src/main/java/io/ballerina/projects/internal/repositories/AbstractPackageRepository.java

```diff
@@ -208,7 +208,7 @@ protected PackageVersion findLatest(List<PackageVersion> packageVersions) {
         return latestVersion;
     }
 
-    protected PackageVersion getLatest(PackageVersion v1, PackageVersion v2) {
+    public PackageVersion getLatest(PackageVersion v1, PackageVersion v2) {
```

Comment:
It is wrong to make this public IMO. We need to move these methods to utility classes. We can do this later anyway!